### ISSUE TITLE
Authentication Not Working

### ISSUE THREAD

**Issue by 123outerme** [Legend-of-iPhoenix/TheButton#34]
I `Ctrl+R`'d, resetting the webpage just to make sure every file was updated, and I became unable to re-authorize my browser. I would login via Google, and the page showed up, but had no leaderboard, and the button did nothing. It didn't ask for a username either. Closing and re-entering somehow fixed it.

**Comment by josheewa** [comment]
Closing and re-opening sometimes is the only way that it properly updates everything. @123outerme please don't click 😝 
***jcgter777 runs

**Comment by ticalc-travis** [comment]
I think I noticed that too a couple of times, though I assumed at first maybe there was some browser glitch or quirk (I have some aggressive script-blocking in place because so many sites do annoying things with them, and that often interferes unless I adjust the settings very carefully). But I had already whitelisted the appropriate scripts when it happened, and I've seen it once each on my desktop and tablet browser, I think.

If I see it again on the desktop, I'll try to remember to check the JS console and see if anything shows up.
